chain/network/src/test_utils.rs

```diff
@@ -237,6 +258,8 @@ impl Handler<StopSignal> for PeerManagerActor {
     }
 }
 
+/// Ban peer for unit tests.
+/// Calls `try_ban_peer` in `PeerManagerActor`.
```

Comment:
shouldn't we have #[cfg(feature = "test_features")] on basically whole file?

Reply:
Some of the tests run without `test_features`  feature. So we can't do that unless we move all tests under `test_features` flag.